Sources/Danger/Danger.swift

```diff
@@ -80,7 +78,9 @@ private func dumpResultsAtExit(_ runner: DangerRunner, path: String) {
             encoder.outputFormatting = .prettyPrinted
             let data = try encoder.encode(dumpInfo.danger.results)
 
-            if !FileManager.default.createFile(atPath: dumpInfo.path, contents: data, attributes: nil) {
+            if !FileManager.default.createFile(atPath: dumpInfo.path,
+                                               contents: data,
+                                               attributes: nil) {
                 dumpInfo.danger.logger.logError("Could not create a temporary file for the Dangerfile DSL at: \(dumpInfo.path)")
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/Danger.swift;
  Line: 84;
-->

- :warning: Line should be 120 characters or less: currently 128 characters (`line_length`)